test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -87,6 +87,30 @@ public async Task GivenMoreThanTenPatients_WhenSearchedWithSortParam_ThenPatient
             }
         }
 
+        [Fact]
+        [HttpIntegrationFixtureArgumentSets(dataStores: DataStore.SqlServer)]
+        public async Task GivenPatientsWithMissingBirthDates_WhenSearchedWithSortParamAndMissingModifierFalse_ThenThosePatientsShouldNotReturn()
+        {
+            var tag = Guid.NewGuid().ToString();
+            var patients = await CreatePaginatedPatientsWithMissingBrithDates(tag);
+
+            var returnedResults = await GetResultsFromAllPagesAsync($"Patient?_tag={tag}&birthdate:missing=false&_sort=birthdate");
+
+            Assert.Equal(10, returnedResults.Count());
+        }
+
+        [Fact]
+        [HttpIntegrationFixtureArgumentSets(dataStores: DataStore.SqlServer)]
+        public async Task GivenPatientsWithMissingBirthDates_WhenSearchedWithSortParamAndNoMissingModifier_ThenAllPatientsShouldReturn()
+        {
+            var tag = Guid.NewGuid().ToString();
+            var patients = await CreatePaginatedPatientsWithMissingBrithDates(tag);
+
+            var returnedResults = await GetResultsFromAllPagesAsync($"Patient?_tag={tag}&_sort=birthdate");
+
+            Assert.Equal(12, returnedResults.Count());
+        }
```

Comment:
I would put them in a single test.

Reply:
Do you see any disadvantage with separation? Now I changed those to parameter based, having them separate is clear.

Reply:
It is not what had in mind. I would just have single test inside which I would test 2 queries, so it clear that they rely on the same functionality and the same data.

Reply:
I like that. So, we can show different behavior on same set of data. I modified the code to include it in single test.